cmd/execution/main.go

```diff
@@ -161,6 +178,38 @@ func main() {
 			pendingBlocks = buffer.NewPendingBlocks() // for following main chain consensus
 			return nil
 		}).
+		Component("Block data uploader", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+			if enableBlockDataUpload {
+
+				logger := node.Logger.With().Str("component_name", "block_data_uploader").Logger()
```

Comment:
```suggestion
				logger := node.Logger.With().Str("component_name", "block_data_uploader").Str("gcp_bucket", gcpBucketName).Logger()
```

Reply:
Bucket name doesn't change, I don't think will get anything extra from logging this but bloating logs